server/src-rsr/initialise.sql

```diff
@@ -616,10 +617,16 @@ CREATE OR REPLACE FUNCTION
                   session_variables := NULL;
       END;
     END IF;
+    BEGIN
+      trace_context := current_setting('hasura.tracecontext');
```

Comment:
Can you use the same `IF .. ELSE` pattern as used for `hasura.user`? (Trapping exceptions is slow in Postgres and since Postgres >= 9.6 supports a way to fetch a session variable's value without raising an exception)

Reply:
Done, thanks!